setup.py

```diff
@@ -12,7 +12,7 @@
 install_requires = [
     "numpy>=1.22.0",
     "pandas>=1.3.0",
-    "pytorch-lightning>=1.6",
```

Comment:
I see these changes, along with the changes in pre-commit were also made in #112 will this cause any issues with the merges?